extensions/nova-act/package.json

```
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nova-act",
+  "title": "Nova Act",
+  "description": "Automates browser tasks using Amazon's Nova Act agent",
+  "icon": "extension-icon.png",
+  "author": "rick_lc_liu",
+  "categories": [
+    "Web",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Nova Act",
+      "subtitle": "Automates browser tasks using AI",
+      "description": "Does some task for you",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "pythonPath",
+      "type": "textfield",
+      "title": "Python Interpreter Path",
+      "placeholder": "/usr/local/bin/python3.12",
+      "description": "Specify the full path to a Python 3.10+ interpreter. You can run `which python` to find the path",
```

```suggestion
      "description": "Specify the full path to a Python 3.10+ interpreter. You can run `which python3` to find the path",
```

Should it be like this?